spec/intro.dd

```diff
@@ -10,59 +10,58 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
+$(H2 Conceptual models)
 
-$(P The process of compiling is divided into multiple phases. Each phase has no
-dependence on subsequent phases. For example, the scanner is not perturbed by
-the semantic analyzer. This separation of the passes makes language tools like
-syntax directed editors relatively easy to produce. It also is possible to
-compress D source by storing it in $(SINGLEQUOTE tokenized) form.)
+$(H3 Environments)
 
-$(OL
-        $(LI $(B source character set)$(BR)
+$(P The data-processing-system in which a D program is translated from source to executable format
+will be called the translation environment and the data-processing-system in which the translated
+D program is executed will be called the execution environment. The translation environment and
```

Comment:
I would suggest highlighting terms like `translation environment` and `execution environment` when first defining them, so they stand out as definitions.  Might even be worth explicitly defining a macro to highlight such definitions.